index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
@dvicj About the html file if you push this to the main branch Mallory's file will be removed, but her file is complete and more structured and it also has the link to tailwind.css.
So I think it's better that you copy her html file and add the POKE CARDS section to it, then push your html file.